content/10-managing-databases/04-introduction-to-OLAP-OLTP.mdx

```diff
@@ -0,0 +1,64 @@
+---
+title: 'Introduction to OLAP & OLTP'
```

Comment:
I just recently came across [this recommendation](https://readabilityguidelines.co.uk/grammar-points/ampersands/) which I'd never seen before, so I thought I'd point it out.  No strong feelings either way on my end though because I know we use it elsewhere.